Handler/Feed.hs

```diff
@@ -24,10 +24,10 @@ feedFromComments (Entity siteId site) comments = do
     render <- getUrlRender
 
     rssFeedText Feed
-        { feedAuthor      = siteRssAuthor site
-        , feedTitle       = siteRssTitle site
-        , feedDescription = siteRssDescription site
-        , feedLanguage    = siteRssLanguage site
+        { feedAuthor      = siteName site
+        , feedTitle       = "Comments on " <> siteName site
+        , feedDescription = toHtml $ "Comments on " <> siteName site
```

Comment:
Sure, I just didn't know if `toHtml` meant "take this literal string and mark it as HTML" or "escape HTML characters in this literal string."